config/resources/government.json

```diff
@@ -0,0 +1,44 @@
+{
+  "version": "0.1",
+  "prefixes": {
+    "kans": "http://kanselarij.vo.data.gift/core/",
+    "skos": "http://www.w3.org/2004/02/skos/core#",
+    "ext": "http://mu.semte.ch/vocabularies/ext/"
+  },
+  "resources": {
+    "government-fields": {
+      "new-resource-base": "http://themis.vlaanderen.be/beleidsveld/",
+      "class": "kans:Beleidsveld",
+      "attributes": {
+        "prefLabel": {
+          "type": "string",
+          "predicate": "skos:prefLabel"
+        }
+      },
+      "relationships": {
+        "hasGovernmentDomain": {
+          "predicate": "ext:heeftBeleidsDomein",
+          "target": "government-domain",
```

Comment:
Target moet dezelfde key zijn als het object dat deze resource specifieert. In dit geval dus `government-domains`. Tenzij er op het `government-domain` model expliciet een andere `name` gezet wordt (zie https://github.com/mu-semtech/mu-cl-resources#overview-of-the-keys), maar dat is hier niet het geval.